extensions/wikia/RobotsTxt/tests/WikiaRobotsTest.php

```diff
@@ -214,13 +214,10 @@ public function testCustomRobotsRulesMultipleNamespaces() {
 
 	public function testAllowedSpecialPages() {
 		$this->mockGlobalVariable( 'wgWikiaEnvironment', WIKIA_ENV_PROD );
+		$this->mockGlobalVariable( 'wgRobotsTxtBlockedWiki', false );
 		$this->mockGlobalVariable( 'wgRobotsTxtCustomRules', null );
 
-		$robotsTxtMock = $this->getRobotsTxtMock();
-		$pathBuilderMock = $this->getPathBuilderMock();
-
-		$wikiaRobots = new WikiaRobots( $pathBuilderMock );
-		$wikiaRobots->configureRobotsBuilder( $robotsTxtMock );
+		$robotsTxtMock = $this->getWikiaRobotsTxt();
 
 		$this->assertTrue( $this->isSpecialPageAllowed( $robotsTxtMock, 'Forum' ) );
 		$this->assertTrue( $this->isSpecialPageAllowed( $robotsTxtMock, 'Sitemap' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Sitemap" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)